string model_name                 # name of the Unity Model to be deleted
---
bool success                      # return true if deletion is successful
string status_message             # comments if available
